Jojoucsd/oop.html

```diff
@@ -0,0 +1,28 @@
+Constructor often use "this" refer to the parent function or object. And this is directional and pointing to the associate function and "this" also must be within the function to make the scope of the variable working. Examples like :
```

Comment:
This file doesn't look like html to me.  If it's just notes maybe use .txt or .md